src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-form/rgw-user-form.component.spec.ts

```diff
@@ -118,54 +110,4 @@ describe('RgwUserFormComponent', () => {
       expect(resp.quotaMaxSize).toBeTruthy();
     });
   });
-
-  describe('userIdValidator', () => {
-    it('should validate user id (1/3)', () => {
```

Comment:
Using your suggestion of the clone PR it makes sense to keep the tests because they test the final functionality. These tests here explicitly test the validator implementation and because of this they are obsolete.

I will add your suggested tests to test the form and validation.

Reply:
Done